.env.development

```diff
@@ -17,3 +17,11 @@ TENANT_ID=not-a-real-secret
 
 SIDEKIQ_WEB_UI_USERNAME=not-a-real-secret
 SIDEKIQ_WEB_UI_PASSWORD=not-a-real-secret
+
+ANALYTICS_USERNAME=analytics_user
+ANALYTICS_PASSWORD=analytics_user_password
+
+kubectl create secret generic rds-postgresql-instance-output \
+    --from-literal=analytics_user=analytics_user \
+    --from-literal=analytics_password='QjAnGA3spw5b3o2' \
+    --namespace=laa-submit-crime-forms-dev
```

Comment:
is this here by accident?

Reply:
yes this part is a WIP as I can't get it to work... :(

Reply:
I think your issue is that `rds-postgresql-instance-output` already exists as a secret auto-created by cloud platform. You're not going to be able to re-create it. I think you'd have to create a new secret in AWS secret manager or add the new keys to the  existing `app-secrets` secret that's there.